Makefile

```diff
@@ -120,7 +120,13 @@ unit: $(CARGO_TARGET_DIR)
 	$(CARGO) test
 
 .PHONY: integration
+# The NETAVARK envvar is used by bats to specify path to a netavark binary.
+ifdef NETAVARK
+$(info Skipping build as NETAVARK binary is specified)
+integration: $(CARGO_TARGET_DIR)
+else
 integration: $(CARGO_TARGET_DIR) examples
+endif
```

Comment:
This causes failures in `500-plugin.bats` these tests are hard coded to look for `./targets/release/examples/`, so we require the examples step.